vibi-dpu/src/github/prs.rs

```diff
@@ -78,12 +78,12 @@ pub async fn get_pr_info_github(repo_owner: &str, repo_name: &str, access_token:
         &base_url, repo_owner, repo_name, pr_number
     );
     println!("[get_pr_info_github] URL: {:?}", &url);
-
     let client = get_client();
     let response_result = client
         .get(&url)
         .header("Authorization", format!("Bearer {}", access_token))
         .header("Accept", "application/json")
+        .header(USER_AGENT, "Vibinex code review Test App")
```

Comment:
Why add "Test App" here? Can't (or shouldn't) we use the same header for our test and prod environments?